public/js/app/components/MemoTable.jsx

```diff
@@ -1,51 +1,49 @@
 import React from 'react'
+import { seasonStr } from '../util/helpers'
+import Table from './Table'
 
 const row = (translation, courseCode, language, courseMemo) => {
-  const { courseOffering, isPdf, memoName, memoVersionsAndUrls } = courseMemo
+  const { semester, courseOffering, isPdf, memoName, memoVersionsAndUrls } = courseMemo
+  return [
+    seasonStr(translation.course_short_semester, semester),
+    courseOffering,
+    <ul className="link-list">
+      {memoName && <li key={memoName}>{memoName + ':'}</li>}
+      {memoVersionsAndUrls.map((v, index) => (
```

Comment:
I'm a fan of _not_ having one-character variables, can we find a good name for `v`, that tells us something more?